src/actions/guides/deploying/ubuntu.cr

```diff
@@ -124,6 +124,15 @@ class Guides::Deploying::Ubuntu < GuideAction
     crystal run tasks.cr -- db.migrate
     ```
 
+    > Note: You may have to specify the `DATABASE_URL` and any other evironment
+    > variables your app needs to migrate. Only `DATABASE_URL` needs to be
+    > real. The rest can be blank. You can add them before running crystal like
+    > so:
+    >
+    > ```bash
+    > API_KEY= SUPPORT_EMAIL= DATABASE_URL=postgresql://<username>:<password>@127.0.0.1/<appname>_production crystal run tasks.cr -- db.migrate
```

Comment:
I'm a bit confused by the SUPPORT_EMAIL and API_KEY. Are those just examples?

Maybe a good example would be `SENGRID_API_KEY`?

Reply:
Yup they are examples of custom environment variables in an app. Curiously I do have `SENGRID_API_KEY` which is required for app boot, but not migration! I can't think of any other built in ENV vars that would make better examples, but I'm all ears.

Reply:
Maybe PORT? This isn't a big deal though so don't feel too much pressure. I think this was an awesome change even left as-is.

Also just FYI. I've been a bit anxious/stressed and am sometimes being overly abrupt in my communication which can come off demanding. So if I do that, I'm sorry in advance and am working on it, but LMK because I definitely don't mean that

Reply:
I had the same deal with `PORT` as with `SENGRID_API_KEY`. What if I add a little more to the explanation like
> For example, if your app needs both the `API_KEY` and `SUPPORT_EMAIL` environment variable:

Sorry to hear that you're anxious/stressed. A lot of that going around these days. Your comments were A-OK as as I'm concerned!

Reply:
> our comments were A-OK as as I'm concerned!

Phew!

Yeah I like the idea of explaining. That should do the trick!

Reply:
https://github.com/luckyframework/website/pull/314